.github/workflows/publish-swaggerhub.yaml

```diff
@@ -61,39 +50,38 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
 
-      - name: Install Swagger CLI
+      - name: Install tools
```

Comment:
It seems you have not rebased onto `main` before working on this. Please revert all your changes of `publish-swaggerhub.yaml` and `publish.yaml` to the current `main` branch.

Reply:
I thought I had, but apparently there was some problem there. I'll look into it when I'm back.